src/components/Badge/test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+import Badge from './index';
+
+
+describe('Badge', () => {
+
+
+    it('Renders correctly', () => {
+        const tree = renderer.create(<Badge><div>child</div></Badge>).toJSON();
+
```

Comment:
Should there be an extra space here, and also on line 32.

Reply:
I was thinking no, but I went ahead and added double spacing to the test file for consistency sake.